### PR TITLE
feat(filman.cc): add presence

### PR DESCRIPTION
**Pull request by xKubuuus**
**Describe the changes in this pull request:**
added `filman.cc` richpresence

<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

and more!
</details>

### CODE REVIEW

**Review by Slowlife01 — CHANGES_REQUESTED** [review]
Revert the changes to yarn.lock file

**Comment by xKubsoneQ** [comment]
updated

File: websites/F/filman.cc/presence.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+const presence = new Presence({
+	clientId: "966334543789424781"
+});
+function browsingTimestamp() {
+	return Math.floor(Date.now() / 1000);
+}
```
Comment (Slowlife01) [inline websites/F/filman.cc/presence.ts:6]:
```suggestion
}),
browsingTimestamp = Math.floor(Date.now() / 1000);
```

**Review by Slowlife01 — CHANGES_REQUESTED** [review]

File: websites/F/filman.cc/presence.ts (hunk 1 above)
Comment (Slowlife01) [reply websites/F/filman.cc/presence.ts]: You have to edit the line 8th too

**Comment by EncryptedDev** [comment]
Revert the changes to `yarn.lock`

**Comment by xKubsoneQ** [comment]
but how i can do it? only commit it to my repo with copied yarn.lock file from this repo?

**Comment by xKubsoneQ** [comment]
nvm, edited

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Comment by EncryptedDev** [comment]
Thank you for your PR! 🎉

**Pull request merged**